apps/extension/src/routes/popup/home/index.tsx

```diff
@@ -27,6 +28,11 @@ export const popupIndexLoader = async (): Promise<Response | PopupLoaderData> =>
 
   if (!password) return redirect(PopupPath.LOGIN);
 
+  const messages = await localExtStorage.get('messages');
+
+  if (messages.find(msg => msg.status === MessageStatus.PENDING))
+    return redirect(PopupPath.ACTIVE_MESSAGE);
```

Comment:
Why are we re-using the index route of popup.html? Shouldn't connected sites just be its own route?

Reply:
When you click on the `Connect` button after you are already connected, it creates a new normal popup window. Feels incorrect.